src/client/src/StreamState.c

```diff
@@ -138,6 +139,15 @@ STATUS defaultStreamStateTransitionHook(
             pKvsRetryStrategy->pRetryStrategy != NULL &&
             pKvsRetryStrategyCallbacks->executeRetryStrategyFn != NULL, STATUS_SUCCESS);
 
+    if(pKvsRetryStrategyCallbacks->getCurrentRetryAttemptNumberFn != NULL) {
+        if((countStatus = pKvsRetryStrategyCallbacks->getCurrentRetryAttemptNumberFn(pKvsRetryStrategy, &pKinesisVideoStream->diagnostics.streamApiCallRetryCount)) != STATUS_SUCCESS) {
+            DLOGW("Failed to get retry count. Error code: %08x", countStatus);
+        }
+        else {
+            DLOGD("Client state machine retry count: %llu", pKinesisVideoStream->diagnostics.streamApiCallRetryCount);
```

Comment:
UINT32 format specifier should be just `%lu`

Reply:
Oops. Fixed in new commit.